R/run_stress_test.R

```diff
@@ -331,9 +336,12 @@ st_read_agnostic <- function(dir, start_year) {
   return(out)
 }
 
-st_process <- function(data, asset_type, company_exclusion) {
+st_process <- function(data, asset_type, company_exclusion, scenario_data) {
```

Comment:
scneario_data is a list entry in data here

Reply:
updated